chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchBasicDetails.java

```diff
@@ -264,6 +265,11 @@
      */
     private String svapNoticeUrl;
 
+    /**
+     * Direct reviews that were conducted against this listing or its developer.
+     */
+    private List<DirectReview> directReviews = new ArrayList<DirectReview>();
```

Comment:
Can you also add the boolean directReviewsAvailable field? That way the user knows if they got 0 DRs because there were 0 or because they weren't available (i.e. Jira is down). Regular listing details has this field if you want to look.